config/toml.go

```diff
@@ -483,6 +480,15 @@ create_empty_blocks_interval = "{{ .Consensus.CreateEmptyBlocksInterval }}"
 peer_gossip_sleep_duration = "{{ .Consensus.PeerGossipSleepDuration }}"
 peer_query_maj23_sleep_duration = "{{ .Consensus.PeerQueryMaj23SleepDuration }}"
 
+#######################################################
+###         Storage Configuration Options           ###
+#######################################################
+
+# Set to false to ensure ABCI responses are persisted. ABCI responses are
+# required for /block_results RPC queries, and to reindex events in the
+# command-line tool.
```

Comment:
Can we mention that setting this to true will save considerable disk space (or something similar)

Reply:
Does the new comment seem reasonable?